src/backend/termwiz.rs

```diff
@@ -221,3 +220,18 @@ impl From<Color> for ColorAttribute {
         }
     }
 }
+
+impl From<(usize, usize)> for Rect {
+    fn from((cols, rows): (usize, usize)) -> Rect {
+        Rect::new(0, 0, u16_max(cols), u16_max(rows))
+    }
+}
+
+fn u16_max(i: usize) -> u16 {
+    let max = u16::max_value();
+    if i > usize::from(max) {
+        max
+    } else {
+        i as u16
+    }
+}
```

Comment:
Replace with `u16::try_from(i).unwrap_or(u16::MAX);` (probably obvious enough to just inline into the methods that use it)

Reply:
Much better, thank you!